CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v0.3.0
+
+* Removed `for_vcpu` argument from `signal::register_signal_handler` and
+`signal::validate_signal_num`. Users now have full control which signals can
+be used for their threads.
```

Comment:
The users had full control before this change too, the syntax was just different:
```suggestion
`signal::validate_signal_num`. Users can now pass absolute values for all valid signal numbers.
```

Reply:
They didn't actually. At least not without "cheating". In the previous version, setting "for_vcpu" to `true` would restrict signal numbers to [SIGRTMIN, SIGRTMAX]. You could argue that one could use the function for setting up a signal handler for a vcpu but set `for_vcpu` to `false` but that's "cheating" 😛 

Either way, I'm not attached to this wording, I'm fine to change it, just wanted to give more context.